packages/repository/README.md

```diff
@@ -222,263 +306,22 @@ coercion. The following types are supported out of box.
-
-// Using the Mixin
-class MyApplication extends RepositoryMixin(Application) {}
-
-// ProductRepository will be bound to key `repositories.ProductRepository`
-const app = new MyApplication({repositories: [ProductRepository]});
-// CategoryRepository will be bound to key `repositories.CategoryRepository`
-app.repository(CategoryRepository);
-```
-
-### Compose repositories and controllers in a context
-
-```ts
-async function main() {
-  // Create a context
-  const ctx = new Context();
-
-  // Mock up a predefined repository
-  const repo = new DefaultCrudRepository(Note, ds);
-
-  // Bind the repository instance
-  ctx.bind('repositories.noteRepo').to(repo);
-
-  // Bind the controller class
-  ctx.bind('controllers.MyController').toClass(NoteController);
-
-  // Resolve the controller
-  const controller: NoteController = await ctx.get('controllers.MyController');
-
-  // Create some notes
-  await controller.create({title: 't1', content: 'Note 1'});
-  await controller.create({title: 't2', content: 'Note 2'});
-
-  // Find notes by title
-  const notes = await controller.findByTitle('t1');
-  return notes;
-}
-
-// Invoke the example
-main().then(notes => {
-  // It should print `Notes [ { title: 't1', content: 'Note 1', id: 1 } ]`
-  console.log('Notes', notes);
-});
-```
-
-### Mix in a repository into the controller (To be implemented)
-
-This style allows repository methods to be mixed into the controller class
-to mimic LoopBack 3.x style model classes with remote CRUD methods. It blends
-the repository responsibility/capability into the controller.
-
-```ts
-import {EntityCrudRepository} from '../../src/repository';
-import {Customer} from '../models/customer';
-import {repository} from "../../src/decorator";
-
-/**
- * Use class level @repository decorator to mixin repository methods into the
- * controller class. Think about @repository as a shortcut to @mixin(...)
- */
-// Style 1
-// Create a repository that binds Customer to mongodbDataSource
-@repository(Customer, 'mongodbDataSource')
-// Style 2
-// Reference a pre-configured repository by name. This is close to LoopBack
-// 3.x model-config.json
-// @repository('myCustomerRepository')
-export class CustomerController {
-  // find() will be mixed in
-}
-```
-
-## Declare pre-defined repositories in JSON/YAML (To be implemented)
-
-Repositories can be declared in JSON/YAML files as follows:
-
-server/repositories.json
-```json
-{
-  "customerRepo": {
-    "dataSource": "mysql",
-    "model": "Customer",
-    "settings": {}
-  }
-}
-```
 ## Related resources
+
 - https://martinfowler.com/eaaCatalog/repository.html
 - https://msdn.microsoft.com/en-us/library/ff649690.aspx
 - http://docs.spring.io/spring-data/data-commons/docs/2.0.0.M3/reference/html/#repositories
 
 ## Contributions
 
-- [Guidelines](https://github.com/strongloop/loopback-next/wiki/Contributing##guidelines)
+- [Guidelines](https://github.com/strongloop/loopback-next/blob/master/docs/site/DEVELOPING.md)
```

Comment:
Change this to point to `https://github.com/strongloop/loopback-next/blob/master/docs/CONTRIBUTING.md`